web/src/pages.rs

```diff
@@ -26,5 +26,13 @@ pub async fn user_dashboard(t: Template, user: User) -> TemplatedPage {
         p { "Welcome, " (user.user_name) }
 
         a href="/logout" { "Logout" }
+
+        h3 {  "Insert Article" }
+        form method="post" action="/insert_articles" {
```

Comment:
Ah so we made this plural, but lets also drop the `insert` prefix!

```suggestion
        form method="post" action="/articles" {
```